### PR TITLE
Re-enable Evergreen tests

### PR DESCRIPTION
**Pull request by hlwarriner**
The Evergreen trunk nightly job succeeded yesterday.

b/270605628

### CODE REVIEW

**Review by kaidokert — APPROVED** [review]
lgtm if the builds go green

**Comment by hlwarriner** [comment]
This is blocked by an issue that's preventing us from updating the packages on the Google Update/Download servers for Cobalt trunk. Hopefully that will be resolved soon and then we can merge this.

**Pull request merged**
